webapp/src/store/actions.ts

```diff
@@ -23,23 +23,24 @@ const actions = wrapActions({
       commit('addBeneficiary', bnf);
     }
   },
-  async createUser({ commit }, { phone, password }: { phone: string; password: string }) {
-    const user = await Users.createUser({ phone, password });
-    await Users.login({ phone, password });
+  async createUser({ commit }, { phone, password, googleIdToken }: { phone: string; password: string; googleIdToken: string }) {
+    const user = await Users.createUser({ phone, password, googleIdToken });
+    await Users.login({ phone, password, googleIdToken });
     commit('setUser', user);
     if (user) {
       router.push({ name: 'home' });
     }
   },
-  async signUserIn({ commit }, { phone, password }: { phone: string; password: string }) {
-    const user = await Users.login({ phone, password });
+  async signUserIn({ commit }, { phone, password, googleIdToken }: { phone: string; password: string; googleIdToken: string }) {
+    const user = await Users.login({ phone, password, googleIdToken });
```

Comment:
I still don't understand why the user needs all three credentials (i.e., phone, password, and googleIdToken) to be logged in

Reply:
They are not all needed at the same time. This was just to cover all possible cases and use only one function. In reality the only possible combinations are phone + pw, phone + google and google.